tests/Bekk.Pact.Consumer.Tests/Builders/PactBuilderTests.cs

```diff
@@ -56,7 +56,7 @@ public void Dispose()
             }
         }
 
-        [Fact]
```

Comment:
Et forslag til neste gang er å legge inn `Skip` i `Fact`-attributtet.

Reply:
Ja, bra tips. Vet du hva som er issuet med tester på myget? Ser ut som det plutselig breaket. 
Tester å bygge på circleci nå.

Reply:
Nei, det har vært en hodepine. Den har kommet og gått. Noe er galt inne i httplistener-implementasjonen.

Reply:
Ser ut til å funke fint i circleci. Så vi kan jo bruke det :)

Reply:
Ja, klarte å reprodusere det lokalt nå. Får: System.Net.Sockets.SocketException: Address already in use
noen ganger, andre ganger ikke.

Reply:
Nei, det er ikke den jeg snakker om. 
Den får du fordi du ikke disposer ordentlig i testene.

Reply:
hm, har ikke gjort noen endringer der som jeg klarer å finne.

Reply:
Kan også skje om du debugger tester og avbryter midt i, vil jeg tro. (Fiksen er å bytte port i testene).